src/Pages/TabsPage.tsx

```diff
@@ -0,0 +1,26 @@
+import { useParams } from 'react-router-dom';
+import { Tabs } from '../components/Tabs';
+import { Tab } from '../types/Tab';
+
+type Props = {
+  tabs: Tab[];
+};
+
+export const TabsPage: React.FC<Props> = ({
+  tabs,
+}) => {
+  const { tabId = 'Tab-0' } = useParams();
+
+  return (
+    <div className="section">
+      <div className="container">
+        <h1 className="title">Tabs page</h1>
+
+        <Tabs
+          tabs={tabs}
+          selectedTabId={`Tab-${tabId}`}
+        />
```

Comment:
Do this into the Tabs component
So, i recommend wrapping if you have more than 2 attributes, or your string length has more than 80

```suggestion
        <Tabs tabs={tabs} selectedTabId={tabId} />
```